installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1398,13 +1398,13 @@ INSERT IGNORE INTO `oauth_user_role_mapping`(`userRoleId`,`userId`,`roleId`,`cli
 /* Display and Update Fields */
 TRUNCATE TABLE cf_pac_updatable_fields;
 INSERT IGNORE INTO cf_pac_updatable_fields  (resourceType,displayFields,updatableFields) VALUES
- ('all_list','_resourceid,tags.Application,tags.Environment,_entitytype',null),
+ ('all_list','_docid,_resourceid,tags.Application,tags.Environment,_entitytype',null),
  ('all_taggable','_resourceid,tags.Application,tags.Environment,_entitytype,targetType,accountid,accountname,region',null),
  ('all_vulnerable','_resourceid,tags.Application,tags.Environment,_entitytype,accountid,accountname,region',null),
  ('all_patchable','_resourceid,tags.Application,tags.Environment,_entitytype',null);
 
-
-INSERT IGNORE INTO OmniSearch_Config (SEARCH_CATEGORY,RESOURCE_TYPE,REFINE_BY_FIELDS,RETURN_FIELDS) VALUES ('Assets','All','accountname,region,tags.Application,tags.Environment,tags.Stack,tags.Role','_resourceid,searchcategory,tags[],accountname,_entitytype');
+TRUNCATE table OmniSearch_Config;
```

Comment:
Should this statement be part of migration script. Once the table is dropped there is not of this line?

Reply:
@plyubich This is a bug, as this table has no primary key for each redeploy inserting the data into this table. Saasdev has 20k rows. This may cause the latency issues in global search. That is the reason I am deleting the rows before inserts. Once this goes to tenants, we can remove the truncate command and add some primary keys to this table.

Reply:
Right. So why not have a separate migration script that you will throw away after deployment and keep the repo clean?